parm/aero/berror/staticb_bump.yaml

```diff
@@ -5,19 +5,21 @@ saber central block:
     io:
       data directory: &staticb_aero_dir $(DATA)/berror
       files prefix: 'nicas_aero'
+      overriding universe radius file: cor_aero_universe_radius
     drivers:
       multivariate strategy: univariate
+      read universe radius: true
       read local nicas: true
-  saber outer blocks:
-  - saber block name: StdDev
-    read: 
-      model file: 
-        datetime: '2016-06-30T00:00:00Z'
-        set datetime on read: true
-        filetype: fms restart
-        psinfile: true
-        datapath: Data/staticb_aero/
-        filename_core: 20160630.000000.stddev.fv_core.res.nc
-        filename_trcr: 20160630.000000.stddev.fv_tracer.res.nc
-        filename_cplr: 20160630.000000.stddev.coupler.res
-        date: '2016-06-30T00:00:00Z'
+saber outer blocks:
+- saber block name: StdDev
+  read: 
+    model file: 
+      datetime: '2021-02-27T00:00:00Z'
+      set datetime on read: true
+      filetype: fms restart
+      psinfile: true
+      datapath: $(DATA)/berror
+      filename_core: 20210227.000000.stddev.fv_core.res.nc
```

Comment:
We need to figure out how to make these dates generic and filled in at runtime based on the config file(s). But that can be a separate PR.

Reply:
Yes, I was thinking about that. There will be several different dates for staticB files available.